recipes/denovogear/build.sh

```diff
@@ -0,0 +1,5 @@
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+
+cp -R ./* $outdir/
```

Comment:
Can you please explicitly copy the binaries by name? This way, we avoid to spam $PREFIX/bin, also in future versions.